apps/prairielearn/package.json

```diff
@@ -57,13 +57,13 @@
     "@prairielearn/zod": "workspace:^",
     "@pyroscope/nodejs": "^0.4.5",
     "@socket.io/redis-adapter": "^8.3.0",
-    "@trpc/client": "^10.45.2",
-    "@trpc/server": "^10.45.2",
+    "@trpc/client": "^11.0.1",
+    "@trpc/server": "^11.0.1",
```

Comment:
This has a migration guide: https://trpc.io/docs/migrate-from-v10-to-v11. I don't know the TRPC interface to know how much this affects PL, but I assume you took care of it.